### PR TITLE
[Feat] Moved docs to seperate repository

### PR DESCRIPTION
**Pull request by dema-trading-ai**
We've moved the documentation to another seperate repository to prevent overlap

### CODE REVIEW

**Review by pvdklei — APPROVED** [review]

File: data/tradingmodule.py [hunk 1]
```diff
@@ -146,6 +145,7 @@ def open_trade(self, ohlcv: Series) -> None:
             print("[INFO] Budget is running low, cannot buy")
             return
 
+        print("OPENING TRADE")
```
Comment (noudcorten) [inline data/tradingmodule.py:148]: @dema-trading-ai Should this be removed?

File: requirements.txt [hunk 2]
```diff
@@ -6,3 +6,4 @@ six==1.15.0
 ta-lib==0.4.19
 pytest==6.2.2
 tqdm==4.58.0
+# qtpylib==1.5.82
```
Comment (noudcorten) [inline requirements.txt:9]: @dema-trading-ai Should this be removed as well? A bit sloppy to have comments.

Comment (dema-trading-ai) [reply requirements.txt]: Solved.

File: data/tradingmodule.py (hunk 1 above)
Comment (dema-trading-ai) [reply data/tradingmodule.py]: Solved.

**Pull request merged**